05-classification.Rmd

```diff
@@ -436,26 +436,48 @@ show the calculation of distance for each of the 5 closest points.
 |`r round(tab[5,1],2) `              |`r round(tab[5,2],2) `        | $\sqrt{(0 - (-1.16))^2 + (3.5 - 4.04)^2} =$ `r round(neighbours[5, "Distance"],2)` | `r tab[5, "Class"]`|
 -----------------     ----------------- ----------------- 
 
+The result of this computation shows that 3 of the 5 nearest neighbours to our new observation are
+malignant (`M`); since this is the majority, we classify our new observation as malignant. 
+These 5 neighbours are circled in Figure \@ref(fig:05-multiknn-3).
+
+```{r 05-multiknn-3, echo = FALSE, fig.cap="Scatter plot of concavity versus perimeter with 5 nearest neighbours circled"}
+perim_concav + annotate("path",
+  x = new_point[1] + 1.4 * cos(seq(0, 2 * pi,
+    length.out = 100
+  )),
+  y = new_point[2] + 1.4 * sin(seq(0, 2 * pi,
+    length.out = 100
+  ))
+)
+```
+</center>
+
 
-**More than two explanatory variables**
+**More than two predictor variables**
 
-Although the above description is directed toward two explanatory variables /
-predictors, exactly the same K-nearest neighbour algorithm applies when you
-have a higher number of explanatory variables (i.e., a higher-dimensional
-predictor space).  Each explanatory variable/predictor can give us new
+Although the above description is directed toward two predictor variables, 
+exactly the same K-nearest neighbour algorithm applies when you
+have a higher number of predictor variables (i.e., a higher-dimensional
+predictor space).  Each predictor variable may give us new
 information to help create our classifier.  The only difference is the formula
 for the distance between points. In particular, let's say we have $m$ predictor
 variables for two observations $u$ and $v$, i.e., 
 $u = (u_{1}, u_{2}, \dots, u_{m})$ and
 $v = (v_{1}, v_{2}, \dots, v_{m})$.
```

Comment:
Why did we switch from x & y to u & v? I think this might confuse learners. Also, if we have to use new letters, u and v are less easily to visually discriminate between than most other letter pairings in the alphabet.

Reply:
we couldn't use x and y twice because x,y are the coordinates in the first example, and u,v are the observation names in the second. now i've just changed both examples to use a, b for observation names.